delta-lake/delta-20x/src/main/scala/org/apache/spark/sql/delta/rapids/delta20x/GpuOptimisticTransaction.scala

```diff
@@ -71,7 +71,7 @@ class GpuOptimisticTransaction
    * @param deltaLog The Delta Log for the table this transaction is modifying.
    * @param rapidsConf RAPIDS Accelerator config settings
    */
-  def this(deltaLog: DeltaLog, rapidsConf: RapidsConf)(implicit clock: Clock) {
+  def this(deltaLog: DeltaLog, rapidsConf: RapidsConf)(implicit clock: Clock) = {
```

Comment:
Is it enforced in 2.13 but also works in 2.12?

I suggest to push such changes incrementally, ahead of the big PR. We should also put a scalastyle check in place preventing such code to be reintroduced while we don't have a 2.13 build as a mandatory merge check.